integration/cosmosgen/bank_module_test.go

```diff
@@ -15,6 +15,8 @@ import (
 )
 
 func TestBankModule(t *testing.T) {
+	t.Skip()
```

Comment:
I will try to figure out the issue with these set of tests before the new release and if so we can enable them.

Reply:
great!! thanks